README.md

```diff
@@ -14,6 +14,7 @@ This integration will add the most important sensors of your Nest Protect device
 
 - Only Google Accounts are supported, there is no plan to support legacy Nest accounts
 - When Nest Protect (wired) occupancy is triggered, it will stay 'on' for 10 minutes. (API limitation)
+- Only *cookie authentication* is supported as Google removed the API key authentication method. This means that you need to login to the Nest website at least once to generate a cookie. This cookie will be used to authenticate with the Nest API. The cookie will be stored in the Home Assistant configuration folder and will be used for future requests.

```

Comment:
Can we mention that when an user logs out / changes their password, that they need to reauthenticate? Is there a limit to the session currently, or will this work indefinitely?

Reply:
Done, It seems that the current implementation updates the uptime every 10 min or so, which will keep the session and cookie alive